### DISCUSSION TITLE
Will there be inline script in 3.0?

### DISCUSSION THREAD

**Discussion by iamdarkle** [orestbida/cookieconsent#404] [Q&A] [answered]
I was wondering if we will have an inline script to run 3.0, I haven't seen anything or at least I haven't found it.

**Comment by orestbida** [comment, marked as answer]
You can use an inline script just like in v2:

```html
<script defer src="cookieconsent.umd.js"></script>
<script>
    window.addEventListener('load', () => {
        CookieConsent.run({ ... });
    });
</script>
```

Make sure you're using the `umd` variant.

**Reply by iamdarkle** [reply]
Oh I see, thank you very much the code I used for 2.0 although I tried to adapt it didn't work, now it works fine, you can mark it as solved 🙂
